source/opt/debug_info_manager.h

```diff
@@ -45,24 +42,21 @@ class DebugInfoManager {
     return !(lhs == rhs);
   }
 
-  // Clones DebugDeclare or DebugValue for a local variable whose result id is
-  // |orig_var_id|. Set Variable operand of the new DebugDeclare as
-  // |new_var_id|. Return the new DebugDeclare or DebugValue. If this is the
-  // first DebugDeclare or DebugValue, keep it in |local_var_id_to_dbgdecl_|.
-  Instruction* CloneDebugDeclare(uint32_t orig_var_id, uint32_t new_var_id);
+  // Analyzes OpenCL.DebugInfo.100 instruction |dbg_inst|.
+  void AnalyzeDebugInst(Instruction* dbg_inst);
 
-  // Returns id of new DebugInlinedAt. Its Line operand is the line number
-  // of |line| if |line| is not nullptr. Otherwise, its Line operand
+  // Creates new DebugInlinedAt and returns its id. Its line operand is the
+  // line number of |line| if |line| is not nullptr. Otherwise, its line operand
   // is the line number of lexical scope of |scope|. Its Scope and Inlined
-  // operands are Scope and Inlined of |scope|. Note that this function puts
-  // the new DebugInlinedAt into the tail of debug instructions.
+  // operands are Scope and Inlined of |scope|.
   uint32_t CreateDebugInlinedAt(const Instruction* line,
                                 const DebugScope& scope);
 
-  // Creates a new DebugInfoNone instruction and returns it. In addition,
-  // insert the new DebugInfoNone instruction before the head of debug
-  // instructions.
-  Instruction* CreateDebugInfoNone();
+  // If |debug_info_none_inst_| is not a nullptr, returns it. Otherwise,
```

Comment:
No need to mention the internal implementation of the class.  You can simply say, "Returns a DebugInfoNone instruction."